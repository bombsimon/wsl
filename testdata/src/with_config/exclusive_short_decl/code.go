package testpkg

func fn() {
	a := 1
	b := 2
	c := 3

	a = 1
	b = 1
	c = 1

	d := 1
	a = 3  // want "missing whitespace decreases readability"
	e := 4 // want "missing whitespace decreases readability"
}
