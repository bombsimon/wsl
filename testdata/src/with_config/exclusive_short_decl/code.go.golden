package testpkg

func fn() {
	a := 1
	b := 2
	c := 3

	a = 1
	b = 1
	c = 1

	d := 1

	a = 3  // want `missing whitespace above this line \(invalid statement above assign\)`

	e := 4 // want `missing whitespace above this line \(invalid statement above assign\)`
}
